detectionmetrics/utils/metrics.py

```diff
@@ -160,3 +160,51 @@ def get_accuracy(self) -> Tuple[np.ndarray, float]:
         )
         acc = np.sum(correct_per_class) / np.sum(total_per_class)
         return acc_per_class, acc
+
+    def get_precision(self) -> np.ndarray:
```

Comment:
Return type is not `np.ndarray` anymore. It should be `Tuple[np.ndarray, float]` (it also has to be updated in the docstring). Same for `get_recall`.